packages/react-scripts-plugin-typescript/src/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const {
+  pushExtensions,
+  pushExclusiveLoader,
+} = require('react-dev-utils/plugins');
+
+function apply(config, { path, paths }) {
```

Comment:
Just curious: is there any reason this isn't just `const path = require('path')`, but passed into the plugin instead?